src/init.cpp

```diff
@@ -800,6 +831,63 @@ void InitLogging()
  */
 bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 {
+    // ********************************************************* Step 0: download bootstrap
```

Comment:
Does this mean it runs on every start? as its part of the App Init

Reply:
yes